### PR TITLE
translation to pt_BR

### PR DESCRIPTION
**Pull request by murilodbva**
translation to pt_BR #

### CODE REVIEW

File: plugins/generic/recommendByAuthor/locale/pt_BR/locale.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * plugins/generic/recommendByAuthor/locale/en_US/locale.xml
```
Comment (mtub) [inline plugins/generic/recommendByAuthor/locale/pt_BR/locale.xml]: should be 
`plugins/generic/recommendByAuthor/locale/pt_BR/locale.xml/pt_BR/locale.xml`

File: plugins/generic/recommendByAuthor/locale/pt_BR/locale.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  *
+  * Copyright (c) 2014-2016 Simon Fraser University Library
+  * Copyright (c) 2003-2016 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  * Localization information: https://pkp.sfu.ca/wiki/index.php?title=Translating_OxS
+  * Localization strings.
+  -->
+ 
+<locale name="en_US" full_name="U.S. English">
```
Comment (mtub) [inline plugins/generic/recommendByAuthor/locale/pt_BR/locale.xml]: `<locale name="pt_BR" full_name="Português (Brasil)">`

File: plugins/generic/recommendByAuthor/locale/pt_BR/locale.xml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+	<message key="plugins.generic.recommendByAuthor.displayName">Artigos Recomendados por Autor</message>
+	<message key="plugins.generic.recommendByAuthor.description">Este plugin insere uma lista de artigos do mesmo autor na página de resumo do artigo.</message>
+    
+    <message key="plugins.generic.recommendByAuthor.heading">Artigos mais lidos do mesmo autor (s)</message>
```
Comment (mtub) [inline plugins/generic/recommendByAuthor/locale/pt_BR/locale.xml]: Are you sure about the blank between autor and (s)?

**Comment by murilodbva** [comment]
@mtub , I guess now the description is ok.
Sorry for the silly mistake.  :/

**Comment by murilodbva** [comment]
See if you can see my correction, please.

**Comment by mtub** [comment]
Thank you!

**Pull request merged**